jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java

```diff
@@ -102,19 +102,20 @@ public void playerBottomSheetTest() {
     }
 
     @Test
-    public void listBottomSheetTest(){
+    public void listBottomSheetTest() {
         listBottomSheet.is().hidden();
 
         listBottomSheetButton.click();
         listBottomSheet.is().displayed();
-        listBottomSheet.openInOptionsList().get(1).find(".v-list-item__title").has().text("Keep");
-        listBottomSheet.openInOptionsList().get(1).find("img").click();
+        listBottomSheet.getOptionTitle(1, "Keep");
```

Comment:
what the reason to keep text by index? You can implement isOptionTitlesPresented() and verify into it that all test titles are presented and verify return value in the test.

Reply:
list of test values can be defined in test level or in page object